### ISSUE TITLE
[Bug]: Missing reading first character from top to bottom of the detailsList while accessibility testing.

### ISSUE THREAD

**Issue by RituShinde** [microsoft/fluentui#30230]
### Library

React Northstar / v0 (@fluentui/react-northstar)

### System Info

```shell
System:
    OS: Windows 10 10.0.19045
    CPU: (8) x64 Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz   
    Memory: 5.30 GB / 15.68 GB
  Browsers:
    Chrome: 120.0.6099.131
    Edge: Spartan (44.19041.3636.0), Chromium (120.0.2210.91)
    Internet Explorer: 11.0.19041.3636
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/p/sandbox/wispy-cache-yfkc7p?file=%2Fsrc%2Findex.tsx%3A491%2C1

## Actual Behavior
JAWS Reader reading the content without reading the first letter.
## Expected Behavior
JAWS Reader should read the content including reading the first letter.

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by micahgodbolt** [comment]
please fix CSB
